### DISCUSSION TITLE
Minecraft Forge 1.20.1 Server does not start

### DISCUSSION THREAD

**Discussion by BlueTreeMotion** [pterodactyl/panel#5094] [Support]
Hi. I am trying to use Forge with Pterodactyl. Sometimes it works fine and sometimes it doesn´t work, allthough i used the same starting parameters for the server.
I need help to fix this issue, so i could install forge servers without any problems for me and others on my pterodactyl.
`[Pterodactyl Daemon]: Updating process configuration files...
[Pterodactyl Daemon]: Ensuring file permissions are set correctly, this could take a few seconds...
container@pterodactyl~ Server marked as starting...
[Pterodactyl Daemon]: Pulling Docker container image, this could take a few minutes to complete...
Pulling from pterodactyl/yolks 
Status: Image is up to date for ghcr.io/pterodactyl/yolks:java_17 
Digest: sha256:7851212e425a09d3d4cc369a8337621ccf59adba8f1182b595881e4f1d1cfe04 
[Pterodactyl Daemon]: Finished pulling Docker container image
container@pterodactyl~ java -version
openjdk version "17.0.11" 2024-04-16
OpenJDK Runtime Environment Temurin-17.0.11+9 (build 17.0.11+9)
OpenJDK 64-Bit Server VM Temurin-17.0.11+9 (build 17.0.11+9, mixed mode, sharing)
container@pterodactyl~ java -Xms128M -Xmx0M -Dterminal.jline=false -Dterminal.ansi=true -jar server.jar
Error: Unable to access jarfile server.jar
container@pterodactyl~ Server marked as offline...
[Pterodactyl Daemon]: ---------- Detected server process in a crashed state! ----------
[Pterodactyl Daemon]: Exit code: 1
[Pterodactyl Daemon]: Out of memory: false
[Pterodactyl Daemon]: Updating process configuration files...
[Pterodactyl Daemon]: Ensuring file permissions are set correctly, this could take a few seconds...
container@pterodactyl~ Server marked as starting...
[Pterodactyl Daemon]: Pulling Docker container image, this could take a few minutes to complete...
Pulling from pterodactyl/yolks 
Status: Image is up to date for ghcr.io/pterodactyl/yolks:java_17 
Digest: sha256:7851212e425a09d3d4cc369a8337621ccf59adba8f1182b595881e4f1d1cfe04 
[Pterodactyl Daemon]: Finished pulling Docker container image
container@pterodactyl~ java -version
openjdk version "17.0.11" 2024-04-16
OpenJDK Runtime Environment Temurin-17.0.11+9 (build 17.0.11+9)
OpenJDK 64-Bit Server VM Temurin-17.0.11+9 (build 17.0.11+9, mixed mode, sharing)
container@pterodactyl~ java -Xms128M -Xmx0M -Dterminal.jline=false -Dterminal.ansi=true -jar server.jar
Error: Unable to access jarfile server.jar
container@pterodactyl~ Server marked as offline...
[Pterodactyl Daemon]: ---------- Detected server process in a crashed state! ----------
[Pterodactyl Daemon]: Exit code: 1
[Pterodactyl Daemon]: Out of memory: false
[Pterodactyl Daemon]: Aborting automatic restart, last crash occurred less than 60 seconds ago.`

![Screenshot 2024-05-08 131042](https://github.com/pterodactyl/panel/assets/14014418/b682dd07-a565-4439-91ce-c10ee4f855d8)
![Screenshot 2024-05-08 131051](https://github.com/pterodactyl/panel/assets/14014418/b74f81f2-83d8-4b52-9aea-dde8cd868ba1)

**Comment by JaniszewskiM** [comment]
Hi,
I know it's an old topic, but maybe it will help others. 

Pterodactyl checks whether unix_args.txt is in the files, if it is not available, the server starts in the "old" way. The quickest way is to go to libraries/net/minecraftforge/forge/<forge_version>, copy the file and put it in the main container folder

**Comment by hidenf901** [comment]
Hi,
I know it's an old topic, but maybe it will help others.

Pterodactyl checks whether unix_args.txt is in the files, if it is not available, the server starts in the "old" way. The quickest way is to go to libraries/net/minecraftforge/forge/<forge_version>, copy the file and put it in the main container folder

**Reply by MrSazid123** [reply]
Bro It Doesn't Work
<img width="1021" height="702" alt="image" src="https://github.com/user-attachments/assets/19b26a3c-4886-4732-9c53-e5518bf44a35" />
